Library/ENV/scm/git

```diff
@@ -10,8 +10,8 @@ fi
 exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
 #!/usr/bin/env ruby -W0
 
-# This script because we support $GIT, $HOMEBREW_SVN, etc. and Xcode-only
-# configurations. Order is careful to be what the user would want.
+# This script because we support $GIT, $HOMEBREW_SVN, etc., Xcode-only and
```

Comment:
Also, I want to ask whether GIT would be used inside formula compiling? After all, this is also picked up by superenv.